autoPyTorch/pipeline/components/setup/network_embedding/base_network_embedding.py

```diff
@@ -33,19 +33,20 @@ def build_embedding(self, num_input_features: np.ndarray, num_numerical_features
 
     def _get_args(self, X: Dict[str, Any]) -> Tuple[int, np.ndarray]:
```

Comment:
Should we delete this method at all?

Reply:
Hmm, I will get back on this one, after I understand the rest of the code additions and changes if this is not necessary anymore. If not needed, it can be removed. 
Additionally, this function should have had some documentation and comments before, it is hard to read.

Reply:
Okay, so to resolve this we would have to decide on how to handle the preprocessing pipeline. To save time, we have agreed to run it only once per dataset and without hyperparameters(this includes imputing, scaling, and encoding and most importantly not feature preprocessing which still has a choice like (PCA, etc.)). However, we may want to also have hyperparameters for the preprocessing pipeline but transform the data once before training so instead of preprocessing once per dataset, we'll do it once per configuration. We need to decide on this tradeoff. Once this is resolved, I'll correct this function.

Reply:
The behavior for the future will be once per configuration.

Reply:
okay so once the experiments are done, I'll adapt this to the implementation for the preprocessor.